source/applications/advanced/hand_eye_calibration/hand_eye_calibration.py

```diff
@@ -49,14 +49,35 @@ def _enter_robot_pose(index):
     return robot_pose
 
 
+def _perform_calibration(hand_eye_input):
+    """Calibration type user input.
+
+    Args:
+        calibration_type: Type of calibration, eth (for eye-to-hand) or eih (for eye-in-hand)
```

Comment:
Gives impression that the input argument is calibration type. But agrs is really hand_eye_input?

Reply:
Good catch. CI also cathed it. Fixed now in https://github.com/zivid/zivid-python-samples/pull/102/commits/07f30aa366d7f015017777f5076e872fcf549e45